test/integration/targets/gcp_compute_ssl_certificate/tasks/main.yml

```diff
@@ -52,10 +53,11 @@
 - name: create a ssl certificate
   gcp_compute_ssl_certificate:
       name: "{{ resource_name }}"
-      description: |
-        "A certificate for testing. Do not use this certificate in production"
+      description: '"A certificate for testing. Do not use this certificate in production"
+
```

Comment:
What's going on here?

Reply:
Fixed!

Reply:
It still has the mysterious extra lines though!

Reply:
WTF, it does? Definitely didn't a moment ago.

One sec, let me investigate!

Reply:
That was weird. The Magician didn't seem to run with my latest commit?

Anyways, just manually checked and the change went through.